### PR TITLE
feat(i18n): add login/signup translation

### PR DESCRIPTION
**Pull request by Jasmine-liang**

### CODE REVIEW

File: src/components/Forms/EmailSignUpForm/Init.tsx [hunk 1]
```diff
@@ -183,7 +182,12 @@ const Init: React.FC<FormProps> = ({
       type="submit"
       form={formId}
       disabled={!isValid || isSubmitting}
-      text={<Translate id="nextStep" />}
+      text={
+        <FormattedMessage
+          defaultMessage="Next"
+          description="src/components/Forms/EmailSignUpForm/Init.tsx"
```
Comment (robertu7) [inline src/components/Forms/EmailSignUpForm/Init.tsx:188]: make it reusable? `description=""`

File: src/components/Forms/EmailSignUpForm/Password.tsx [hunk 2]
```diff
@@ -153,30 +153,53 @@ const Password: React.FC<FormProps> = ({
   const InnerForm = (
     <Form id={formId} onSubmit={handleSubmit}>
       <Form.Input
-        label={<Translate id="password" />}
+        label={
+          <FormattedMessage
+            defaultMessage="Password"
+            description="src/components/Forms/EmailSignUpForm/Password.tsx"
```
Comment (robertu7) [inline src/components/Forms/EmailSignUpForm/Password.tsx:159]: make it reusable?

File: src/components/Forms/EmailSignUpForm/Password.tsx [hunk 3]
```diff
@@ -153,30 +153,53 @@ const Password: React.FC<FormProps> = ({
...
+          />
+        }
         type="password"
         name="password"
         required
-        placeholder={translate({
-          id: 'enterPassword',
-          lang,
+        placeholder={intl.formatMessage({
+          defaultMessage: 'Password',
```
Comment (robertu7) [inline src/components/Forms/EmailSignUpForm/Password.tsx:166]: Will `Enter Password` be better to understand?

File: src/components/Forms/EmailSignUpForm/Password.tsx [hunk 4]
```diff
@@ -153,30 +153,53 @@ const Password: React.FC<FormProps> = ({
...
+          defaultMessage: 'Enter password again',
+          description: 'src/components/Forms/EmailSignUpForm/Password.tsx',
+        })}
         value={values.comparedPassword}
         error={touched.comparedPassword && errors.comparedPassword}
-        hint={<Translate id="hintPassword" />}
+        hint={
+          <FormattedMessage
+            defaultMessage="Minimum 8 characters. Uppercase/lowercase letters, numbers and symbols are allowed"
+            description="src/components/Forms/EmailSignUpForm/Password.tsx"
```
Comment (robertu7) [inline src/components/Forms/EmailSignUpForm/Password.tsx:200]: ignore the description to make it reusable?

File: src/components/Forms/WalletAuthForm/Connect.tsx [hunk 5]
```diff
@@ -307,23 +312,22 @@ const Connect: React.FC<FormProps> = ({
...
-          label={<Translate id="email" />}
+          label={<FormattedMessage defaultMessage="Email" description="" />}
           type="email"
           name="email"
           required
-          placeholder={translate({
-            id: 'enterEmail',
-            lang,
+          placeholder={intl.formatMessage({
+            defaultMessage: 'Enter Email',
```
Comment (robertu7) [inline src/components/Forms/WalletAuthForm/Connect.tsx:329]: `enterEmail` is `Email` in `text.ts`?

File: src/components/Forms/WalletAuthForm/Connect.tsx [hunk 6]
```diff
@@ -417,7 +427,7 @@ const Connect: React.FC<FormProps> = ({
       type="submit"
       form={formId}
       disabled={isSubmitting || loading || !account}
-      text={<Translate id="nextStep" />}
+      text={<FormattedMessage defaultMessage="Next Step" description="" />}
```
Comment (robertu7) [inline src/components/Forms/WalletAuthForm/Connect.tsx:430]: `"Next"`?

File: src/components/Forms/WalletAuthForm/Select.tsx [hunk 7]
```diff
@@ -297,7 +288,17 @@ const Select: React.FC<FormProps> = ({
...
-            <Translate id={isConnect ? 'loginWithWallet' : 'authEntries'} />
+            isConnect ? (
+              <FormattedMessage
+                defaultMessage="Connect Wallet"
+                description=""
+              />
+            ) : (
+              <FormattedMessage
+                defaultMessage="Enter"
+                description="src/components/Forms/WalletAuthForm/Select.tsx"
```
Comment (robertu7) [inline src/components/Forms/WalletAuthForm/Select.tsx:299]: empty description to make it reusable?

File: src/components/Forms/WalletAuthForm/Connect.tsx (hunk 5 above)
Comment (Jasmine-liang) [reply src/components/Forms/WalletAuthForm/Connect.tsx:329]: Yes

File: src/components/Forms/WalletAuthForm/Connect.tsx (hunk 6 above)
Comment (Jasmine-liang) [reply src/components/Forms/WalletAuthForm/Connect.tsx:430]: "nextStep" in `text.ts` is "Next Step"

**Review by robertu7 — APPROVED** [review]

**Pull request merged**